src/components/Books.js

```diff
@@ -1,10 +1,16 @@
-import React from 'react';
-import { useSelector } from 'react-redux';
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
 import Book from './sub-components/book';
 import AddNewBook from './sub-components/AddNewBook';
+import { getBooksFromServer } from '../redux/books/books';
 
 export default function Books() {
+  const dispatch = useDispatch();
   const books = useSelector((store) => store.books);
+  useEffect(() => {
+    // console.log('data');
```

Comment:
- [ ] **Optional:** Kindly remove commented code. It is not professional to push commented code.